tutorials/quantize.md

```diff
@@ -1,14 +1,32 @@
 # Quantize the model
 
-When `--precision bf16-true` or `--precision 16-true` is used, the model weights will automatically be converted and consume less memory.
-However, this might not be enough for large models or when using GPUs with limited memory.
+This document provides different strategies for quantizing the various models available in Lit-GPT to reduce GPU memory usage, which is useful for running larger models on certain GPU hardware. 
+
+**All the examples below were run on an A100 40GB GPU.**
 
 > **Note**:
 > Quantization is only supported with inference (generate and chat scripts).
 
+
 ### Baseline
```

Comment:
I understand that it was like that before, but other "chapters" uses only double # sign.